docs/src/main/paradox/user/lang/index.md

```diff
@@ -11,3 +12,30 @@ Cloudstate user functions can be implemented in any language that supports gRPC.
 * [Go](go/index.md)
 
 @@@
+
+## Unofficial client libraries
+We are aware of external libraries implementing the **client CloudState protocol** in various other languages.
+
+:warning: **Though they are mentioned here, these libraries are often not directly maintained by the CloudState dev team and are therefore not supported**.
+ 
+Here is a list of other, **non-official** libraries in various other languages:
+
+| Library Name | Language | Event-based support? | CRDT support? |
+|--------------|----------|----------------------|---------------|
+| [scala-support](https://github.com/cloudstateio/cloudstate/tree/master/scala-support/src/main) | Scala | :warning: | :warning: |
+| [kotlin-support](https://github.com/cloudstateio/kotlin-support) | Kotlin | :warning: | :warning: |
+| [cloudstate-csharp](https://github.com/nagytech/cloudstate-csharp)| C# | :heavy_check_mark: | :heavy_check_mark: | 
+| [python-support](https://github.com/marcellanz/cloudstate_python-support/tree/feature/python-support)| Python | | |
+| | Elixir | | |
```

Comment:
You may want to remove implementations that do not currently have a repository like Elixir, F#, Ruby, etc